### DISCUSSION TITLE
Quest internet access - multiple answers

### DISCUSSION THREAD

**Discussion by hermann-san** [streetcomplete/StreetComplete#5053] [General]
Hi, in this quest https://github.com/streetcomplete/StreetComplete/pull/784  I can only select one of the answers. However, a Hotel I know of does have multiple  possibilities for internet access.  E.g. WLAN + LAN. That I can't do in Street Complete. Would it be good to have check-boxes instead of radio button's ?
![grafik](https://github.com/streetcomplete/StreetComplete/assets/7679654/3bb1420a-f571-4436-9dc3-3e83563da5b8)

**Comment by arrival-spring** [comment]
This adds the key `internet_access`, the [wiki page](https://wiki.openstreetmap.org/wiki/Key:internet_access) does not mention adding multiple values. However, there has been some discussion on [the talk page](https://wiki.openstreetmap.org/wiki/Talk:Key:internet_access) that multiple values can be used and separated by a semi-colon. This has [minimal usage](https://taginfo.openstreetmap.org/keys/internet_access#values) so I'm not sure if it would count as established enough to change the quest.

**Reply by hermann-san** [reply]
is 250k entries for WLAN considered as minimal usage? 
Maybe there are going to be more entries for LAN and the others if multiple entries are possible. 
Right now I would select WLAN cause it's the most popular, but it's actually not true that it's only WLAN.

**Reply by mnalis** [reply]
> is 250k entries for WLAN considered as minimal usage?

It think you misunderstood; it is the **combination of multiple values** that has minimal usage (e.g. **50** cases worldwide of `internet_access=wlan;wired`) 

>  Maybe there are going to be more entries for LAN and the others if multiple entries are possible.

Maybe, but that suggestion should be discussed in global OSM community (E.g. OSM [tagging mailing list](https://wiki.openstreetmap.org/wiki/Tagging_mailing_list) and/or [community forum](https://community.openstreetmap.org/c/general/tagging/70/)) , not in StreetComplete-specific discussion (as it is not specific to StreetComplete, nor is StreetComplete really authoritative to decide that). Once such proposition is accepted by OSM community, it can be adopted by StreetComplete too.

>  Right now I would select WLAN cause it's the most popular, but it's actually not true that it's only WLAN.

In the meantime, one can also answer `Other answers` / `Can't say` / `Leave note`. Certainly not the ideal solution, but workable. You can then later find [your unsolved notes](https://my-notes.osm-hr.org) and map them with your preferred general-purpose editor to add any tags you like. But note that as such values are not (currently) commonly expected nor [documented](https://wiki.openstreetmap.org/wiki/Key:internet_access), it is likely that adding such values like `internet_access=wlan;wired` will likely result in data consumers ignoring them like nothing is entered (which is probably not desired outcome).

**Reply by hermann-san** [reply]
@mnalis thanks a lot for your helpful explanation. I will try to bring up this topic in those OSM communities.

**Reply by mnalis** [reply]
> I will try to bring up this topic in those OSM communities.

Thanks @hermann-san ; when you do, please add link to them in this discussion; so interested parties may follow.

**Reply by hermann-san** [reply]
[OSM community forum discussion](https://community.openstreetmap.org/t/using-the-same-key-with-multiple-values/93426/6) on multiple values for internet access

**Reply by hermann-san** [reply]
it looks like the discussion is over but  I'm not sure what the outcome now is. Simply documenting as I suggested?

    multiple connection types can be added by using semi-colon separated values internet_access=wlan;wired

**Reply by arrival-spring** [reply]
Yes, documenting it seems good to me

**Reply by hermann-san** [reply]
documentation done.
@mnalis Can the quest now be changed?

**Reply by mnalis** [reply]
Seems reasonable enough to me (i.e. nothing much more that we can do about that to improve the outcome), so feel free to open the [issue](https://github.com/streetcomplete/StreetComplete/issues) (and mention this discussion in there). 

Now that it was discussed with community and documented, the main potential blocker that I see is low tag usage vs. increased complexity of the quest; but project leader will give his decision about that when issue is opened...

**Reply by hermann-san** [reply]
https://github.com/streetcomplete/StreetComplete/issues/5136
